utils/src/print.c

```diff
@@ -48,27 +49,32 @@ static void print_inode(struct scoutfs_key *key, void *val, int val_len)
 	struct scoutfs_inode *inode = val;
 
 	printf("    inode: ino %llu size %llu version %llu nlink %u\n"
-	       "      uid %u gid %u mode 0%o rdev 0x%x flags 0x%x\n"
-	       "      next_readdir_pos %llu meta_seq %llu data_seq %llu data_version %llu\n"
-	       "      atime %llu.%08u ctime %llu.%08u\n"
-	       "      mtime %llu.%08u\n",
-	       le64_to_cpu(key->ski_ino),
-	       le64_to_cpu(inode->size),
```

Comment:
Don't change all this surrounding code.  I'm certainly not going to squint to see what was changed, and the alignment was right to begin with.

Reply:
I blame VSCode